docs/tutorial/stacking_explanation.md

```diff
@@ -0,0 +1,13 @@
+# Stacking explanation
+
+In the following animation you can see an example how selecting different
+measures and dimensions cause different stacking behaviour in the chart.
+
+First we set the x channel to a dimension and an arbitrary constant value to
```

Comment:
I would point out that the first phase shows that dimensions slice the axis to equal sections.